### ISSUE TITLE
SDK3

### ISSUE THREAD

**Issue by robinwidhede** [bochove/HomeySonoffLan#1]
Is it possible for you to update this to SDK3.

Thanks in advance.

**Comment by bochove** [comment]
I did not know anyone but me was making use of this app.
Just to confirm, you are making use of the plain old Sonoff basic switch with default firmware as sold here:
https://www.aliexpress.com/item/4000390205431.html
In LAN mode and you are capable of providing a fixed IP address to this device in order for this app to function?

I am still making use of this app myself, so as soon as I need to I will get it updated to SDK 3.

**Comment by robinwidhede** [comment]
Yes using it, but now I have a Homey Pro 2023. So I need it updated.
